lib/service/TokenList.dart

```diff
@@ -111,6 +111,17 @@ class TokenList {
           idToAddress[ath[1]]![2]);
     }),
   ];
+
+  static String mapTickerToName(String ticker) {
+    return tokenList.firstWhere((token) => token.ticker == ticker).name;
+  }
+
+  //Ask about this
+  static AssetImage mapAptTypeToImage(String longOrShort) {
+    return longOrShort.toLowerCase() == "long"
+        ? AssetImage('assets/images/apt_noninverted.png')
```

Comment:
The images path should be stored in some constant somewhere where other common UI objects are placed(if any place).

Not perfect but for time being would do it.